src/main/java/ch/njol/skript/expressions/ExprSpawn.java

```diff
@@ -44,7 +44,7 @@
  * @author Peter Güttinger
  */
 @Name("Spawn")
-@Description("The spawnpoint of a world.")
+@Description("The spawning point of a world.")

```

Comment:
Is `spawn point` bad?

Reply:
Not really, spawning point sounds better in my opinion tho

Reply:
It is "spawn point" in Minecraft itself and it is pretty correct lol